specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/assessments.json

```diff
@@ -0,0 +1,250 @@
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SecurityAssessment"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SecurityAssessmentList": {
+      "type": "object",
+      "description": "List of security assessments",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SecurityAssessment"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page."
+        }
+      }
+    },
+    "SecurityAssessment": {
+      "type": "object",
+      "description": "Security assessment on a resource",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SecurityAssessmentProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "SecurityAssessmentProperties": {
+      "type": "object",
+      "description": "Describes properties of an assessment.",
+      "properties": {
+        "resourceDetails": {
+          "$ref": "#/definitions/ResourceDetails"
+        },
+        "displayName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "User friendly display name of the assessment"
+        },
+        "status": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Status of the ",
+          "enum": ["Passed", "Failed", "NotApplicable"],
+          "x-ms-enum": {
+            "name": "Severity",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Passed",
+                "description": "The resource is healthy"
+              },
+              {
+                "value": "Failed",
+                "description": "The resource has a security issue that needs to be addressed"
+              },
+              {
+                "value": "NotApplicable",
+                "description": "Assessment for this resource did not happen"
+              }
+            ]
+          }
+        },
+        "statusReasonCode": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Programmatic code for the reason the assessment result status"
+        },
+        "statusReasonDescription": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Human readable description for the reason the assessment result status"
+        },
+        "additionalData": {

```

Comment:
>additionalData [](start = 9, length = 14)

Are these known ahead of time or completely arbitrary?

Reply:
completely arbitrary,
the set of additional data keys are decided per assessment (we have more than 50 assessments and growing) and currently the set is still changing for some of the assessments.
one day it might be more stable, but we will have more types than we can easily document